build.sbt

```diff
@@ -1416,6 +1416,11 @@ lazy val `language-server-deps-wrapper` = project
     }
   )
 
+lazy val nativeLibsOutDir = TaskKey[File](
+  "nativeLibsOutdir",
+  "Directory where native libraries are extracted to"
+)
+
 lazy val `jna-wrapper` = project
```

Comment:
I think we should remove JNA from engine/ls. The arguments remain the same as https://github.com/enso-org/enso/issues/7178#issuecomment-1634588128